R/utils-schema.R

```diff
@@ -75,6 +79,21 @@ get_schema_valid_versions <- function(branch = "main") {
 #' schema_url <- get_schema_url(config = "tasks", version = "v0.0.0.9")
 #' get_schema(schema_url)
 get_schema <- function(schema_url) {
+  # If the branch is "main", then we can use the stored schemas inside the
+  # package.
+  pieces <- extract_schema_info(schema_url)
+  if (pieces$branch[1] == "main") {
+    ver <- pieces$version
+    cfg <- pieces$config
```

Comment:
```suggestion
    version <- pieces$version
    config <- pieces$config
```
For clarity and to conform with variable names in other functions in hubUtils. Also hear we are dealing with the schema files. Config files are what hub admins generate and are validates by the appropriate schema files